src/vue/forms/CreateFundForm.vue

```diff
@@ -0,0 +1,628 @@
+              </div>
+            </div>
+            <div class="app__form-row create-fund__form-row">
+              <div class="app__form-field">
+                <input-field
+                  white-autofill
+                  type="number"
+                  v-model="form.fundInformation.softCap"
+                  @blur="touchField('form.fundInformation.softCap')"
+                  id="soft-cap"
+                  :label="'create-fund-form.soft-cap' | globalize({
+                    asset: config.DEFAULT_QUOTE_ASSET
+                  })"
+                  :error-message="getFieldErrorMessage(
+                    'form.fundInformation.softCap',
+                    {
+                      from:MIN_AMOUNT,
+                      to:MAX_AMOUNT
+                    }
+                  )"
+                  :disabled="formMixin.isDisabled"
+                />
+              </div>
+            </div>
+            <div class="app__form-row create-fund__form-row">
+              <div class="app__form-field">
+                <input-field
+                  white-autofill
+                  type="number"
+                  v-model="form.fundInformation.hardCap"
+                  @blur="touchField('form.fundInformation.hardCap')"
+                  id="hard-cap"
+                  :label="'create-fund-form.hard-cap' | globalize({
+                    asset: config.DEFAULT_QUOTE_ASSET
+                  })"
+                  :error-message="getFieldErrorMessage(
+                    'form.fundInformation.hardCap',
+                    {
+                      from:form.fundInformation.softCap,
+                      to:MAX_AMOUNT
+                    }
+                  )"
+                  :disabled="formMixin.isDisabled"
+                />
+              </div>
+            </div>
+            <div class="app__form-row create-fund__form-row">
+              <div class="app__form-field">
+                <input-field
+                  white-autofill
+                  v-model="form.fundInformation.baseAssetForHardCap"
+                  @blur="touchField('form.fundInformation.baseAssetForHardCap')"
+                  id="base-asset-for-hard-cap"
+                  type="number"
+                  :label="'create-fund-form.base-asset-hard-cap-to-sell' |
+                    globalize({
+                      asset: form.fundInformation.baseAsset
+                    })"
+                  :error-message="getFieldErrorMessage(
+                    'form.fundInformation.baseAssetForHardCap',
+                    {
+                      from:MIN_AMOUNT,
+                      to:availableForIssuance
+                    }
+                  )"
+                  :disabled="formMixin.isDisabled"
+                />
+              </div>
+            </div>
+            <div class="app__form-row create-fund__form-row">
+              <div class="app__form-field">
+                {{ 'create-fund-form.price' | globalize({
```

Comment:
wrap into a `<p>` and apply a class with font-size to wipe out direct inheritance of body

Reply:
ok